src/diffusers/models/resnet.py

```diff
@@ -329,38 +329,38 @@ def __init__(
             self.conv_shortcut = torch.nn.Conv2d(in_channels, out_channels, kernel_size=1, stride=1, padding=0)
 
     def forward(self, x, temb, hey=False):
```

Comment:
```suggestion
    def forward(self, x, temb):
```

If ok for you @JC-swEng we could remove this here as well (this parameter is never used)